arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -566,153 +572,2046 @@
+        },
+        "EnablePartitioning": {
+          "type": "boolean",
+          "description": "value that indicates whether the queue to be partitioned across multiple message brokers is enabled."
+        },
+        "ForwardDeadLetteredMessagesTo": {
+          "type": "string",
+          "description": "the path to the recipient to which the dead lettered message is forwarded."
+        },
+        "ForwardTo": {
+          "type": "string",
+          "description": "the path to the recipient to which the message is forwarded."
+        },
+        "IsAnonymousAccessible": {
+          "type": "boolean",
+          "description": "a value that indicates whether the message is anonymous accessible."
+        },
+        "LockDuration ": {
+          "type": "string",
+          "format": "duration",
+          "description": "the duration of a peek lock; that is, the amount of time that the message is locked for other receivers. The maximum value for LockDuration is 5 minutes; the default value is 1 minute."
+        },
+        "MaxDeliveryCount ": {
+          "format": "int32",
+          "type": "integer",
+          "description": "the maximum delivery count. A message is automatically deadlettered after this number of deliveries."
+        },
+        "MaxSizeInMegabytes ": {
+          "format": "int64",
+          "type": "integer",
+          "description": "the maximum size of the queue in megabytes, which is the size of memory allocated for the queue."
+        },
+        "MessageCount ": {
+          "format": "int64",
+          "type": "integer",
+          "description": "the number of messages in the queue."
+        },
+        "MessageCountDetails": {
+          "type": "object",
+          "description": "message details for a queue."
+        },
+        "Path": {
+          "type": "string",
+          "description": "the name of the queue."
+        },
+        "RequiresDuplicateDetection": {
+          "type": "boolean",
+          "description": "the value indicating if this queue requires duplicate detection."
+        },
+        "RequiresSession": {
+          "type": "boolean",
+          "description": "a value that indicates whether the queue supports the concept of session."
+        },
+        "SizeInBytes ": {
+          "format": "int64",
+          "type": "integer",
+          "description": "the size of the queue in bytes."
+        },
+        "EntityStatus ": {
+          "type": "string",
+          "description": "Enumerates the possible values for the status of a messaging entity.",
+          "enum": [
+            "Active",
+            "Creating",
+            "Deleting",
+            "Disabled",
+            "ReceiveDisabled",
+            "Renaming",
+            "Restoring",
+            "SendDisabled",
+            "Unknown"
+          ],
+          "x-ms-enum
```

Comment:
this is done correctly. Why is this not done for topicResource and subscriptionResource?